CMakeLists.txt

```diff
@@ -3,10 +3,20 @@ cmake_minimum_required (VERSION 3.0)
 
 project(SWMM)
 
-set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
-set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/lib)
+#set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin)
+#set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib)
```

Comment:
Why was this needed :-p ?

Reply:
It sets the install paths

Reply:
@bemcdonnell I believe this is the change that breaks the Travis build. The install path was set to build/bin now they are in src/bin?